src/v0/destinations/ga4/customMappingsHandler.js

```diff
@@ -0,0 +1,198 @@
+const get = require('get-value');
+const jsonpath = require('rs-jsonpath');
+const {
+  validateEventName,
+  basicValidation,
+  isReservedEventName,
+  addClientDetails,
+  removeReservedParameterPrefixNames,
+  prepareUserConsents,
+  removeInvalidParams,
+  GA4_RESERVED_PARAMETER_EXCLUSION,
+  getGA4CustomParameters,
+  buildDeliverablePayload,
+  GA4_PARAMETERS_EXCLUSION,
+  prepareUserProperties,
+} = require('./utils');
+const { InstrumentationError } = require('@rudderstack/integrations-lib');
+const {
+  removeUndefinedAndNullRecurse,
+  constructPayload,
+  isDefinedAndNotNull,
+  isEmptyObject,
+  removeUndefinedAndNullValues,
+  isHybridModeEnabled,
+  getIntegrationsObj,
+} = require('../../util');
+const { trackCommonConfig, ConfigCategory, mappingConfig } = require('./config');
+
+const findGA4Events = (eventsMapping, event) => {
```

Comment:
This works only for track events. How about handling identify/page/group message types ?

Reply:
Addressed